packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -614,7 +614,7 @@ test(`zcf.makeEmptySeatKit`, async t => {
 test(`zcfSeat from zcf.makeEmptySeatKit - only these properties exist`, async t => {
   const expectedMethods = [
```

Comment:
Line 629 below should also have a `.sort()` to avoid counting on the order produced by `Object.keys`. Here too, why are we testing exact agreement on method names? Also, using `Object.keys` here but `getOwnPropertyNames` elsewhere for exactly the same purpose is pointlessly inconsistent. The difference is that `Object.keys` will skip non-enumerable properties whereas `getOwnPropertyNames` will include them. The test here should probably not be sensitive to whether the property is enumerable, so `getOwnPropertyNames` is probably better. If we should have such tests at all.